docker-compose.yml

```diff
@@ -6,42 +6,66 @@ services:
     container_name: data-node-db
     restart: always
     environment:
-      - MONGO_INITDB_ROOT_USERNAME=${MONGO_INITDB_ROOT_USERNAME}
-      - MONGO_INITDB_ROOT_PASSWORD=${MONGO_INITDB_ROOT_PASSWORD}
-      - MONGO_INITDB_DATABASE=${MONGO_INITDB_DATABASE}
-      - MONGO_USERNAME=${MONGO_USERNAME}
-      - MONGO_PASSWORD=${MONGO_PASSWORD}
+      - MONGO_INITDB_ROOT_USERNAME
+      - MONGO_INITDB_ROOT_PASSWORD
+      - MONGO_INITDB_DATABASE
+      - MONGO_USERNAME
+      - MONGO_PASSWORD
     volumes:
       - database-data:/data/db
+    networks:
+      - nlpsandbox-data-node-internal
     ports:
-      - "${MONGO_PORT}:27017"
+      - "27017:27017"
+    deploy:
+      resources:
+        limits:
+          memory: 1G
 
   data-node:
     image: nlpsandbox/data-node:1.2.1
     build:
       context: server
       dockerfile: Dockerfile
     container_name: data-node
-    # command: uwsgi --ini uwsgi.ini --lazy --http :8080 --processes 4
+    # Uncomment the line `command` if you want to customize any of its
+    # parameters. Replace the following values:
+    # - {processes}: min(2 * max number of concurrent processes that the data
+    #   node should support, 6 * number of CPU cores)
+    #command: uwsgi --ini uwsgi.ini --lazy --http :8080 --processes {processes}
     environment:
-      - SERVER_PROTOCOL=${SERVER_PROTOCOL}
-      - SERVER_DOMAIN=${SERVER_DOMAIN}
-      - SERVER_PORT=${SERVER_PORT}
-      - DB_PROTOCOL=${DB_PROTOCOL}
+      - SERVER_PROTOCOL
+      - SERVER_DOMAIN
+      - SERVER_PORT
+      - DB_PROTOCOL
       - DB_DOMAIN=db
-      - DB_PORT=${DB_PORT}
-      - DB_DATABASE=${DB_DATABASE}
-      - DB_USERNAME=${DB_USERNAME}
-      - DB_PASSWORD=${DB_PASSWORD}
+      - DB_PORT
+      - DB_DATABASE
+      - DB_USERNAME
+      - DB_PASSWORD
     # healthcheck:
     #   test: curl --fail http://localhost:8080/api/v1/healthCheck
     #   interval: 10s
     #   timeout: 5s
     #   retries: 5
+    networks:
+      - nlpsandbox-data-node
+      - nlpsandbox-data-node-internal
     ports:
-      - "${SERVER_PORT}:8080"
```

Comment:
Ca you keep this env variable?

Reply:
@thomasyu888 I have restored this variable and updated the UI address in the README.